.github/actions/run-gradle/action.yml

```diff
@@ -52,15 +67,21 @@ runs:
         TIMEOUT_MINUTES: ${{ inputs.timeout-minutes}}
         TEST_CATALOG: ${{ inputs.test-catalog-path }}
         TEST_TASK: ${{ inputs.test-task }}
+        TEST_RETRIES: ${{ inputs.test-retries }}
+        RUN_NEW_TESTS: ${{ inputs.run-new-tests }}
+        RUN_FLAKY_TESTS: ${{ inputs.run-flaky-tests }}
+        TEST_XML_OUTPUT_DIR: ${{ inputs.test-xml-output }}
       run: |
         set +e
         ./.github/scripts/thread-dump.sh &
         timeout ${TIMEOUT_MINUTES}m ./gradlew --build-cache --continue --no-scan \
         -PtestLoggingEvents=started,passed,skipped,failed \
         -PmaxParallelForks=2 \
-        -PmaxTestRetries=1 -PmaxTestRetryFailures=3 \
-        -PmaxQuarantineTestRetries=3 -PmaxQuarantineTestRetryFailures=0 \
+        -PmaxTestRetries=$TEST_RETRIES -PmaxTestRetryFailures=0 \
         -Pkafka.test.catalog.file=$TEST_CATALOG \
+        -Pkafka.test.run.quarantined=$RUN_NEW_TESTS \
```

Comment:
Could we align the naming? otherwise, we may have some ambiguous terms in the future.